### DISCUSSION TITLE
No `set_auto_operate` and `toggle_auto_operate` sockets for the `large turret hardpoint` in the submarine editor

### DISCUSSION THREAD

**Discussion by Mhlov** [FakeFishGames/Barotrauma#15062] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

When I go into wiring mode in the Submarine Editor, the `set_auto_operate` and `toggle_auto_operate` sockets are missing on the `Large Turret Hardpoint`.

### Reproduction steps

1. Enter the submarine editor
2. Add a `large turret hardpoint`
3. Try to wire it

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Other

### -

submarine editor

### Version

v1.6.19.1 (Unto the Breach Update Hotfix 2)

### Which operating system did you encounter this bug on?

Linux

**Comment by Regalis11** [comment]
Thank you for the report! Fixed in https://github.com/FakeFishGames/Barotrauma-development/commit/38e561376c198f2b8e14f366a58db50809897efc (commit in our private repo).
